openkfw/TruBudget
# ui: Only allow available budgets in subproject

# Changes
When creating a subproject, the projected budget for a subproject can
only be selected from the projected budget of the parent project.
The organization is now a dropdown filled with the values of distinct
organizations in the project's projected budget.
E2E tests are added that test this functionality.

Closes #228

# How to test
Create a project with one or more projected budgets, then open the project and create a subproject - the organization field should be a drop down filled with the organizations from the projected budgets of the parent project.
